app/src/androidTest/java/com/example/jasmiensofiecels/wordly/view/dailyWord/ValidDailyWordActivityTest.java

```diff
@@ -0,0 +1,189 @@
+import android.view.View;
+
+import com.example.jasmiensofiecels.wordly.R;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.assertThat;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.CoreMatchers.is;
+
+
+/**
+ * The purpose of this class is to test the main search functionality of a set of words that are valid and invalid.
+ *
+ * Created by Jasmien Cels on 19/04/2018.
+ */
+@RunWith(AndroidJUnit4.class)
+public class ValidDailyWordActivityTest {
+
+    private String validInput;
+
+    /*
+     * The purpose of the rule is to provide functional testing of a single activity.
+     * This provides access to the activity and its components by launching the activity in
+     * a test environment.
+     */
+    @Rule
+    public ActivityTestRule<DailyWordActivity> rule = new ActivityTestRule<>(DailyWordActivity.class);
+
+    @Before
+    public void initValidSearch() {
+
+        validInput = "android";
+
+        //When
+        onView(withId(R.id.search_word_et)) //get text edit widget
+                .perform(typeText(validInput), closeSoftKeyboard()); //insert valid searchable word
+        onView(withId(R.id.search_fab)).perform(click()); //Action click on the search button
+
+        try {
+            Thread.sleep(500);
+        } catch (Exception e) {
+            Log.d("Error", "thread sleep");
+        }
+    }
+
+
+    @Test
+    //The purpose of this test is to ensure that the definedWord textView gets changed to the searched word
+    public void wordTextView_changedOnSearch() {
+        //Then
+        onView(withId(R.id.defined_word_tv))
+                .check(matches(withText(validInput))); //Searched word is displayed.
+
+    }
+
+    @Test
+    //The purpose of test is to ensure that phoneticSpelling is changed
+    public void lexicalTextView_changedOnSearch() {
+        //Then
+        onView(withId(R.id.word_lexical_category))
+                .check(matches(withText("Noun")));
+    }
+
+    //For some reason, it cannot match the phonetic spelling
+//    @Test
+//    public void phoneticTextView_changedOnSearch() {
+//        //Then
+//        onView(withId(R.id.word_phonetic))
+//                .check(matches(withText("andrɔɪd")));
+//    }
+
```

Comment:
This fails because you don't match the `ˈ` at the beginning of `ˈandroid`. 

Options are `containsString()` or fix the text you're matching:
```
onView(withId(R.id.word_phonetic))
                .check(matches(withText("ˈandrɔɪd")));
```

```
onView(withId(R.id.word_phonetic))
                .check(matches(withText(containsString("andrɔɪd"))));
```